themes/crisp/translations/pt_BR.json

```
@@ -64,21 +64,21 @@
     "errors.404.service": "Serviço não encontrado",
     "views.about.featured.text": "Extensão para o seu navegador",
     "views.about.jumbotron.text.1": "“Termos de Serviço; Não Li” (em inglês “Terms of Service; Didn't Read“, abreviando: ToS;DR) é um projeto iniciado em 2012 para ajudar a corrigir “a maior mentira na web”: quase ninguém realmente lê os termos de serviço com os quais concordamos o tempo todo.",
-    "views.about.jumbotron.text.2": "<strong>Contexto.</strong> A ideia por trás ToS;DR emergiu durante o <a href=\"http://events.ccc.de/camp/2011/wiki/ToS;DR\">Chaos Communication Camp 2011</a> próximo a Berlim, com pessoas do movimento <a href=\"http://unhosted.org\">Unhosted</a> para criar aplicativos web que deem controle aos usuários sobre os seus dados valiosos e privacidade. No verão de 2012, <a href=\"http://hugoroy.eu/index.en.html\">Hugo Roy</a> (<a href=\"http://twitter.com/hugoroyd\">@hugoroyd</a>) iniciou a análise legal e trouxe o projeto à vida. Desde então, mais pessoas se juntaram à equipe e contribuíram pela comunidade de revisores. Por fim, todo o trabalho é transparente e as discussões ocorrem em público. O nosso trabalho é <a href=\"thanks\">financiado</a> por organizações sem fins lucrativos e doações individuais e é disponibilizado como <a href=\"http://fsfe.org/about/basics/freesoftware.html\">software livre</a> e dados abertos.",
+    "views.about.jumbotron.text.2": "<strong>Contexto.</strong> A ideia por trás ToS;DR emergiu durante o <a href=\"http://events.ccc.de/camp/2011/wiki/ToS;DR\">Chaos Communication Camp 2011</a> próximo a Berlim, com pessoas do <a href=\"http://unhosted.org\">Unhosted</a> um movimento voltado para a criação de aplicativos web que deem controle aos usuários sobre os seus dados valiosos e sua privacidade. No verão de 2012, <a href=\"http://hugoroy.eu/index.en.html\">Hugo Roy</a> (<a href=\"http://twitter.com/hugoroyd\">@hugoroyd</a>) iniciou a análise legal e trouxe o projeto à vida. Desde então, mais pessoas se juntaram à equipe e contribuíram pela comunidade de revisores. Por fim, todo o trabalho é transparente e as discussões ocorrem em público. O nosso trabalho é <a href=\"thanks\">financiado</a> por organizações sem fins lucrativos e doações individuais e é disponibilizado como <a href=\"http://fsfe.org/about/basics/freesoftware.html\">software livre</a> e dados abertos.",
```

```suggestion
    "views.about.jumbotron.text.2": "<strong>Contexto.</strong> A ideia por trás ToS;DR emergiu durante o <a href=\"http://events.ccc.de/camp/2011/wiki/ToS;DR\">Chaos Communication Camp 2011</a> próximo a Berlim, com pessoas do <a href=\"http://unhosted.org\">Unhosted</a>, um movimento voltado para a criação de aplicativos web que deem controle aos usuários sobre os seus dados valiosos e sua privacidade. No verão de 2012, <a href=\"http://hugoroy.eu/index.en.html\">Hugo Roy</a> (<a href=\"http://twitter.com/hugoroyd\">@hugoroyd</a>) iniciou a análise legal e trouxe o projeto à vida. Desde então, mais pessoas se juntaram à equipe e contribuíram pela comunidade de revisores. Por fim, todo o trabalho é transparente e as discussões ocorrem em público. O nosso trabalho é <a href=\"thanks\">financiado</a> por organizações sem fins lucrativos e doações individuais e é disponibilizado como <a href=\"http://fsfe.org/about/basics/freesoftware.html\">software livre</a> e dados abertos.",
```

Concordo com as alterações. Faltou uma vírgula ([aposto](https://www.soportugues.com.br/secoes/sint/sint22.php)) que eu também havia esquecido.

---

Boa!